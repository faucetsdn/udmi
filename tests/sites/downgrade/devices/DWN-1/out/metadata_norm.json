{
  "timestamp": "2020-05-01T13:39:07Z",
  "version": "1.4.1",
  "hash": "fafc4eb5",
  "cloud": {
    "auth_type": "RS256"
  },
  "system": {
    "location": {
      "site": "US-SFO-XYY",
      "section": "2-3N8C",
      "position": {
        "x": -111.0,
        "y": 102.3
      }
    },
    "physical_tag": {
      "asset": {
        "guid": "bim://04aEp5ymD_$u5IxhJN2aGi",
        "site": "US-SFO-XYY",
        "name": "AHU-1"
      }
    },
    "min_loglevel": 300
  },
  "localnet": {
    "families": {
      "virtual": {
        "addr": "0x65"
      }
    }
  },
  "pointset": {
    "points": {
      "filter_alarm_pressure_status": {
        "units": "No-units",
        "ref": "BV11.present_value"
      },
      "filter_differential_pressure": {
        "units": "Bars"
      },
      "filter_differential_pressure_sensor": {
        "units": "Degrees-Celsius",
        "ref": "AV12.present_value"
      }
    }
  }
}