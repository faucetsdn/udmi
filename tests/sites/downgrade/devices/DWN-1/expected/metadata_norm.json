{
  "cloud": {
    "auth_type": "RS256"
  },
  "gateway": {
    "target": {
      "family": "vendor"
    }
  },
  "hash": "09a1fede",
  "localnet": {
    "families": {
      "vendor": {
        "addr": "0x65"
      }
    }
  },
  "pointset": {
    "points": {
      "filter_alarm_pressure_status": {
        "ref": "BV11.present_value",
        "units": "No-units"
      },
      "filter_differential_pressure": {
        "units": "Bars"
      },
      "filter_differential_pressure_sensor": {
        "ref": "AV12.present_value",
        "units": "Degrees-Celsius"
      }
    }
  },
  "system": {
    "location": {
      "position": {
        "x": -111.0,
        "y": 102.3
      },
      "section": "2-3N8C",
      "site": "US-SFO-XYY"
    },
    "min_loglevel": 300,
    "physical_tag": {
      "asset": {
        "guid": "bim://04aEp5ymD_$u5IxhJN2aGi",
        "name": "AHU-1",
        "site": "US-SFO-XYY"
      }
    }
  },
  "timestamp": "2020-05-01T13:39:07Z",
  "upgraded_from": "1",
  "version": "1.5.3"
}
